test/Microsoft.Azure.Webjobs.Extensions.Storage.UnitTests/ScenarioTests.cs

```diff
@@ -19,43 +19,43 @@ public class ScenarioTests
         private const string QueueName = "queue";
 
         [Fact]
-        public void BlobTriggerToQueueTriggerToBlob_WritesFinalBlob()
+        public async Task BlobTriggerToQueueTriggerToBlob_WritesFinalBlob()
         {
             // Arrange
-            IStorageAccount account = CreateFakeStorageAccount();
-            IStorageBlobContainer container = CreateContainer(account, ContainerName);
-            IStorageBlockBlob inputBlob = container.GetBlockBlobReference(BlobName);
-            inputBlob.UploadText("15");
+            StorageAccount account = await CreateFakeStorageAccountAsync();
+            CloudBlobContainer container = await CreateContainerAsync(account, ContainerName);
+            CloudBlockBlob inputBlob = container.GetBlockBlobReference(BlobName);
+            await inputBlob.UploadTextAsync("15");
 
             // Act
             RunTrigger<object>(account, typeof(BlobTriggerToQueueTriggerToBlobProgram),
                 (s) => BlobTriggerToQueueTriggerToBlobProgram.TaskSource = s);
 
             // Assert
-            IStorageBlockBlob outputBlob = container.GetBlockBlobReference(OutputBlobName);
+            CloudBlockBlob outputBlob = container.GetBlockBlobReference(OutputBlobName);
             string content = outputBlob.DownloadText();
             Assert.Equal("16", content);
         }
 
-        private static IStorageBlobContainer CreateContainer(IStorageAccount account, string containerName)
+        private static async Task<CloudBlobContainer> CreateContainerAsync(StorageAccount account, string containerName)
         {
-            IStorageBlobClient client = account.CreateBlobClient();
-            IStorageBlobContainer container = client.GetContainerReference(containerName);
-            container.CreateIfNotExists();
+            CloudBlobClient client = account.CreateCloudBlobClient();
+            CloudBlobContainer container = client.GetContainerReference(containerName);
+            await container.CreateIfNotExistsAsync();
             return container;
         }
 
-        private static IStorageAccount CreateFakeStorageAccount()
+        private static async Task<StorageAccount> CreateFakeStorageAccountAsync()
         {
-            var account = new FakeStorageAccount();
+            var account = new XFakeStorageAccount();
```

Comment:
What is XFakeStorageAccount? Naming seems a little strange

Reply:
Agree. I meant to remove the `X` classes. I'll do that now.